repositories/src/main/scala/cromwell/pipeline/datastorage/dto/CMWLInput.scala

```diff
@@ -0,0 +1,10 @@
+package cromwell.pipeline.datastorage.dto
+
+import cromwell.pipeline.model.wrapper.UserId
+
+final case class CMWLInput(
```

Comment:
```suggestion
final case class CromellInput(
```
I think it's better to use a full name here

Reply:
Agree. Fixed.